### PR TITLE
[Doc Issue][Are the instructions to enable sideloading correct][2501097]

### PR DESCRIPTION
**Pull request by v-silakshmi**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1601720358e188b3df84a88716686c8fafeca4e8/msteams-platform%2Fm365-apps%2Fprerequisites.md) | 90 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9a752af4-1f67-4ede-a1ce-c4c76c53fe39) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e0f23420e9a04f761202e79acf36f84ef35bcd9b/msteams-platform%2Fm365-apps%2Fprerequisites.md) | 90 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/baa86979-e107-4a35-9887-ee739f03ac4a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f0d9d9b38ec1b17206dd2883b1f3e56742232a53/msteams-platform%2Fm365-apps%2Fprerequisites.md) | 90 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/239a03fd-010b-4fb7-ae0c-1933dade25be) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d0fb95a11117e0e3b17b15b3a930446c68c39ae8/msteams-platform%2Fm365-apps%2Fprerequisites.md) | 90 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/287efd51-ee2e-4c25-80d3-a1e18a69b1df) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/cd30c016b226d4d3600fab8b4250543313265942/msteams-platform%2Fm365-apps%2Fprerequisites.md) | 91 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/addee16f-043a-402a-b9d5-fbc1f7cacecb) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/7b192f5c16acfb383876cd9e61317c94a46e796e/msteams-platform%2Fm365-apps%2Fprerequisites.md) | 91 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1bc4fbe7-acff-4c57-9d04-89a76c55260f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/121429f74627a916bfc49aa266bc54c339c33f9a/msteams-platform%2Fm365-apps%2Fprerequisites.md) | 91 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a8d97f62-5683-468b-a87f-9590b19ec365) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/7c63bbee1a62fb8a5564ba2e1e8be6f69275d1a5/msteams-platform%2Fm365-apps%2Fprerequisites.md) | 91 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1fe19a0f-b4d1-4641-bdc4-1d973b9de945) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/m365-apps/prerequisites.md [hunk 1]
```diff
@@ -25,15 +25,26 @@ You need a Microsoft 365 developer subscription sandbox tenant to set up your de
 
 You'll also need to enable sideloading for your tenant:
 
-1. Sign in to Microsoft 365 admin center (https://admin.microsoft.com) with your test tenant credentials and select **Teams** from the side panel to open the *Microsoft Teams admin center*.
-1. Select: Teams apps > Manage apps > **Org-wide app settings**.
-1. Under **Custom apps**, toggle on the option *Interaction with custom apps*.
+ 1. Sign in to [Microsoft 365 admin center](https://admin.microsoft.com/Adminportal/Home#/homepage) with your test tenant credentials.
```
Comment (aosolis) [inline msteams-platform/m365-apps/prerequisites.md:28]: Can we point directly to the Teams admin center (admin.teams.microsoft.com)?

File: msteams-platform/m365-apps/prerequisites.md [hunk 2]
```diff
@@ -25,15 +25,26 @@ You need a Microsoft 365 developer subscription sandbox tenant to set up your de
...
-    :::image type="content" source="images/teams-admin-enable-sideloading.png" alt-text="Enable sideloading for custom apps from the Teams admin center":::
+ 1. From the left pane, select **Show all**.
+
+ 1. Under admin centers, select **Teams**. A new tab opens the Microsoft Teams admin center.
+
+ 1. Go to **Teams apps** > **Manage apps**.
+
+ 1. At the top right, select **Org-wide app settings**.
+
+ 1. Under Custom apps, turn on the **Interaction with custom app** toggle.
```
Comment (aosolis) [inline msteams-platform/m365-apps/prerequisites.md:38]: Please check that we have the exact strings here. It's actually "Interaction with custom apps"

File: msteams-platform/m365-apps/prerequisites.md [hunk 3]
```diff
@@ -25,15 +25,26 @@ You need a Microsoft 365 developer subscription sandbox tenant to set up your de
...
+
+    :::image type="content" source="images/teams-admin-enable-sideloading.png" alt-text="The screenshot is an example that enables sideloading for custom apps from the Teams Admin Center":::
+
+> [!NOTE]
+> Apart from Org-wide app settings, App setup policies also allow users to upload custom apps to Teams. If you are unable to upload custom apps after updating the Org-wide app settings, verify if the **Upload custom apps** option is enabled in **App setup policies**. For more information, see [Manage app setup policies](/microsoftteams/teams-app-setup-policies).
```
Comment (aosolis) [inline msteams-platform/m365-apps/prerequisites.md:43]: Why is this a Note? We should document the full process here. If app setup policy is required we should say so.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c39fd1c1a9c57c1cc24ee1f290811cfc339c7f4b/msteams-platform%2Fm365-apps%2Fprerequisites.md) | 89 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ca44f082-4cd8-4f23-86f6-c58a98e75f38) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/m365-apps/prerequisites.md (hunk 3 above)
Comment (v-silakshmi) [reply msteams-platform/m365-apps/prerequisites.md]: Removed the note as suggested.

File: msteams-platform/m365-apps/prerequisites.md (hunk 2 above)
Comment (v-silakshmi) [reply msteams-platform/m365-apps/prerequisites.md]: Suggested changes implemented.

File: msteams-platform/m365-apps/prerequisites.md (hunk 1 above)
Comment (v-silakshmi) [reply msteams-platform/m365-apps/prerequisites.md]: suggested changes implemented.

File: msteams-platform/m365-apps/prerequisites.md [hunk 4]
```diff
@@ -25,15 +25,25 @@ You need a Microsoft 365 developer subscription sandbox tenant to set up your de
...
 
-1. Sign in to Microsoft 365 admin center (https://admin.microsoft.com) with your test tenant credentials and select **Teams** from the side panel to open the *Microsoft Teams admin center*.
-1. Select: Teams apps > Manage apps > **Org-wide app settings**.
-1. Under **Custom apps**, toggle on the option *Interaction with custom apps*.
+ 1. Sign in to [Teams admin center](https://admin.teams.microsoft.com/dashboard) with your test tenant credentials.
 
-    :::image type="content" source="images/teams-admin-enable-sideloading.png" alt-text="Enable sideloading for custom apps from the Teams admin center":::
+ 1. From the left pane, select **Show all**.
+
+ 1. Under admin centers, select **Teams**. A new tab opens the Microsoft Teams admin center.
```
Comment (aosolis) [inline msteams-platform/m365-apps/prerequisites.md:32]: If you're linking to admin.teams.microsoft.com, is this still needed? I thought that went directly tot the teams admin center.

File: msteams-platform/m365-apps/prerequisites.md [hunk 5]
```diff
@@ -25,15 +25,25 @@ You need a Microsoft 365 developer subscription sandbox tenant to set up your de
...
+
+ 1. Go to **Teams apps** > **Manage apps**.
+
+ 1. At the top right, select **Org-wide app settings**.
+
+ 1. Under Custom apps, turn on the **"Interaction with custom app"** toggle.
+
+    :::image type="content" source="images/teams-admin-enable-sideloading.png" alt-text="The screenshot is an example that enables sideloading for custom apps from the Teams Admin Center":::
+
+Apart from Org-wide app settings, App setup policies also allow users to upload custom apps to Teams. If you are unable to upload custom apps after updating the Org-wide app settings, verify if the **Upload custom apps** option is enabled in **App setup policies**. For more information, see [Manage app setup policies](/microsoftteams/teams-app-setup-policies).
```
Comment (aosolis) [inline msteams-platform/m365-apps/prerequisites.md:42]: Why is there an "if" here? If the "Upload custom apps" setting is required, then we should say so directly here.

File: msteams-platform/m365-apps/prerequisites.md (hunk 4 above)
Comment (v-silakshmi) [reply msteams-platform/m365-apps/prerequisites.md:32]: Yes the link is directed to Teams admin center.

File: msteams-platform/m365-apps/prerequisites.md (hunk 5 above)
Comment (v-silakshmi) [reply msteams-platform/m365-apps/prerequisites.md:42]: In this instance: if an user is unable to upload custom apps after updating the org-wide settings, user needs to check if the upload custom apps option is enabled or not in app set up policies.

File: msteams-platform/m365-apps/prerequisites.md (hunk 4 above)
Comment (aosolis) [reply msteams-platform/m365-apps/prerequisites.md:32]: Remove it if it's not actually needed, given that the link is now admin.teams.microsoft.com

File: msteams-platform/m365-apps/prerequisites.md (hunk 5 above)
Comment (aosolis) [reply msteams-platform/m365-apps/prerequisites.md:42]: Ok, I'll be blunt. Remove this "if" hedging, and make it an additional step to enable the "Upload custom apps" setting.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/771cf982c4e6594dd203fe50e932f32bab7fba8c/msteams-platform%2Fm365-apps%2Fprerequisites.md) | 89 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/87c30b3b-ff0d-4613-bace-2597b5814b9c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/m365-apps/prerequisites.md (hunk 4 above)
Comment (v-silakshmi) [reply msteams-platform/m365-apps/prerequisites.md:32]: removed it as suggested.

File: msteams-platform/m365-apps/prerequisites.md (hunk 5 above)
Comment (v-silakshmi) [reply msteams-platform/m365-apps/prerequisites.md]: Removed "if" as advised also added it as an additional step.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c379793b74969b73592bace3735ea631ba4b99aa/msteams-platform%2Fm365-apps%2Fprerequisites.md) | 90 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/63e5a86a-345b-4ac5-a29a-32b3bebe0e42) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e175e4b14827ee547b57fc50f8fc00d3f0bb0601/msteams-platform%2Fm365-apps%2Fprerequisites.md) | 90 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8ab48745-c7f9-43a4-9326-f396b985b199) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4b7bddaefe431108365f315d45cbee24dc9fecbf/msteams-platform%2Fm365-apps%2Fprerequisites.md) | 90 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4203e6dc-463c-4cdd-94e2-c4860e1011ea) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/m365-apps/prerequisites.md [hunk 6]
```diff
@@ -25,15 +25,21 @@ You need a Microsoft 365 developer subscription sandbox tenant to set up your de
...
-    :::image type="content" source="images/teams-admin-enable-sideloading.png" alt-text="Enable sideloading for custom apps from the Teams admin center":::
+ 1. Go to **Teams apps** > **Manage apps**.
+
+ 1. At the top right, select **Org-wide app settings**.
+
+ 1. Under Custom apps, turn on the **"Interaction with custom app"** toggle and save.
+
+    :::image type="content" source="images/teams-admin-enable-sideloading.png" alt-text="The screenshot is an example that enables sideloading for custom apps from the Teams Admin Center":::
+
+ 1. Apart from Org-wide app settings, app setup policies also allow users to upload custom apps to Teams. Enable the **Upload custom apps** option in **App setup policies**. For more information, see [Upload custom apps setting](/microsoftteams/upload-custom-apps).
```
Comment (aosolis) [inline msteams-platform/m365-apps/prerequisites.md:38]: just add the following steps

1. In the left navigation of the Microsoft Teams admin center, go to Teams apps > Setup policies, and then select the Global (Org-wide default) policy.

1. Turn on Upload custom apps, and then select Save.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c4548763b96c3790a925583bdb346d4a646aacb6/msteams-platform%2Fm365-apps%2Fprerequisites.md) | 90 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/16ed3d47-18fd-44bd-a052-35be1ed96481) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/m365-apps/prerequisites.md (hunk 6 above)
Comment (v-silakshmi) [reply msteams-platform/m365-apps/prerequisites.md]: Added the steps as suggested.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f75157b5290f2d252c0b550c9f09b3f4916897d1/msteams-platform%2Fm365-apps%2Fprerequisites.md) | 90 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1b8cea2d-3d2b-4880-9b23-c68b15c188eb) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/7b711e47c2455033a17b553091a97cd96a6f991f/msteams-platform%2Fm365-apps%2Fprerequisites.md) | 90 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a1de490a-3b97-4809-a5fc-22c3dd72365a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/32686f22140e1f6c83054e12935fcbfbdb314691/msteams-platform%2Fm365-apps%2Fprerequisites.md) | 90 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0035e8de-6da8-4ed7-b87b-341ec2bd36d0) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by v-habibaumme** [comment]
Hi @aosolis,

Request you to please review and approve.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9502b47a9708c8514bebc01be937b7f82b19e473/msteams-platform%2Fm365-apps%2Fprerequisites.md) | 90 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0f2535b0-6682-4801-9f8e-9cd3866c4955) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by aosolis — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**